runelite-client/src/main/java/net/runelite/client/plugins/grandexchange/GrandExchangePlugin.java

```diff
@@ -428,6 +430,49 @@ public void onWidgetLoaded(WidgetLoaded event)
 		}
 	}
 
+	@Subscribe
+	public void onScriptCallbackEvent(ScriptCallbackEvent event)
+	{
+		if (!event.getEventName().equals("setGETitle") || !config.showTotal())
+		{
+			return;
+		}
+
+		int total = 0;
+		String strGETitle = " (";
+
+		for (int i = 0; i < MAX_OFFERS; i++)
+		{
+			SavedOffer savedOffer = getOffer(i);
```

Comment:
Did you try `client.getGrandExchangeOffers()` instead? That would be more robust.

Reply:
Definitely better, fixed.